provider/github/utils.go

```diff
@@ -67,7 +67,8 @@ func castHash(sha1 *string) plumbing.Hash {
 func castPullRequest(r *lookout.RepositoryInfo, pr *github.PullRequest) *lookout.ReviewEvent {
 	pre := &lookout.ReviewEvent{}
 	pre.Provider = Provider
-	pre.InternalID = string(pr.GetID())
+	pre.InternalID = fmt.Sprintf("%v", pr.GetID())
```

Comment:
usually, people do it using `strconv.FormatInt` though I don't know if there is much difference